addon/MetadataStore.js

```diff
@@ -191,7 +191,7 @@ MetadataStore.prototype = {
       description: aRow.description,
       media_url: aRow.media && aRow.media.url,
       expired_at: aRow.expired_at,
-      metadata_source: aRow.metadata_source || "Embedly",
+      metadata_source: aRow.metadata_source || "MetadataService",
```

Comment:
I'd really rather not have this here.  I know it's just replacing the one we already had but MetadataStore shouldn't know about this, this should be in preview provider.